### PR TITLE
[operator] Fetch ImageStreams tag from CR

### PR DESCRIPTION
**Pull request by lgarciaaco**

### CODE REVIEW

**Comment by lgarciaaco** [comment]
Fixes https://github.com/syndesisio/syndesis/issues/3305
Fixes https://github.com/syndesisio/syndesis/issues/6330

**Comment by lgarciaaco** [comment]
@zregvart how was that again about the test failing?

File: install/operator/pkg/syndesis/action/install.go [hunk 1]
```diff
@@ -245,6 +260,37 @@ func (a *installAction) Execute(ctx context.Context, syndesis *v1alpha1.Syndesis
 	return err
 }
 
+// Checks that the tags from syndesis components is valid beetween the supported versions
+func checkTags(context *generator.Context) error {
+	c := version.NewConstrainGroupFromString(fmt.Sprintf(">=%s,<%s", context.TagMinor, context.TagMajor))
+	if c.Match(version.Normalize(context.Syndesis.Spec.Components.Server.Tag)) == false {
+		return fmt.Errorf("tag for server[%s] component is not supporter, versions supported are between [%s] and [%s]",
```
Comment (zregvart) [inline install/operator/pkg/syndesis/action/install.go]: I've noticed the typo, which made me thing that perhaps we can make this a bit clearer
```suggestion
		return fmt.Errorf("tag for server[%s] component is not valid, tag should have a value between [%s] (inclusive) and [%s] (exclusive)",
```

Similarly for the messages below.

**Comment by lgarciaaco** [comment]
@zregvart I reopened the PR, you changes are included at https://github.com/syndesisio/syndesis/pull/6346, and the rebase with master

**Pull request closed without merging**